flow/util/distributed.py

```diff
@@ -0,0 +1,897 @@
+'''
```

Comment:
Could you change the file name to something like AutoTuner.py (or autotuner.py)? (AutoTuner is currently our proposed name, so I hope it appears in the file name) Also, I hope you could also mention AutoTuner in the internal header comments of the file.

Reply:
I have renamed the script since the goal was to have one script that manages distributed workloads. Currently AutoTuner and sweeping. Also, it can be expanded in the future.

Reply:
Thank you for your reply. I understand. Could you address the below comments?

Reply:
Yes, I will. Do you have any other code comments or are these the only ones for you to approve the PR? If there will be more change requests I will wait for them and make the changes in one go.